tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -440,11 +442,11 @@ public void testApplicationMetricsJSON() {
                 .body("'metricTest.test1.count'", equalTo(1))
 
                 .body("'metricTest.test1.countMeA'", equalTo(1))
-
+                .body("'metricTest.test1.countMeB'", equalTo(666666))
```

Comment:
should this be returning 1?  the `@Counted` is counting the number of hits to the method.